ui/src/dashboards/components/OverlayControls.js

```diff
@@ -3,51 +3,51 @@ import classnames from 'classnames'
 
 import ConfirmButtons from 'shared/components/ConfirmButtons'
 
-import graphTypes from 'hson!shared/data/graphTypes.hson'
-
-const OverlayControls = props => {
-  const {
-    onCancel,
-    onSave,
-    selectedGraphType,
-    onSelectGraphType,
-    isSavable,
-  } = props
-  return (
-    <div className="overlay-controls">
-      <h3 className="overlay--graph-name">Cell Editor</h3>
-      <div className="overlay-controls--right">
-        <p>Visualization Type:</p>
-        <ul className="nav nav-tablist nav-tablist-sm">
-          {graphTypes.map(graphType =>
-            <li
-              key={graphType.type}
-              className={classnames({
-                active: graphType.type === selectedGraphType,
-              })}
-              onClick={() => onSelectGraphType(graphType.type)}
-            >
-              {graphType.menuOption}
-            </li>
-          )}
-        </ul>
-        <ConfirmButtons
-          onCancel={onCancel}
-          onConfirm={onSave}
-          isDisabled={!isSavable}
-        />
-      </div>
+const OverlayControls = ({
+  onCancel,
+  onSave,
+  isDisplayOptionsTabOpen,
+  onSelectDisplayOptions,
+  isSavable,
+}) =>
+  <div className="overlay-controls">
+    <h3 className="overlay--graph-name">Cell Editor</h3>
+    <ul className="nav nav-tablist nav-tablist-sm">
+      <li
+        key="queries"
+        className={classnames({
+          active: !isDisplayOptionsTabOpen,
+        })}
+        onClick={onSelectDisplayOptions(false)}
```

Comment:
personally i'd rather see this wrapped in an anonymous function rather than wrapping the called function, but that's a personal style thing